Yoast/wordpress-seo
# add fallback selectors for imageUrl, title and description

## Summary


This PR can be summarized in the following changelog entry:

* Add a fallback selectors.

## Relevant technical choices:

* After a check from @omarreiss, we decided to add only fallbacks for the title, description and imageUrl. Author and siteTitle is set on the php side.
The title and description are collected from analysisData because they were not easy accessible in the snippet preview. 

## Test instructions

This PR can be tested by following these steps:

* Run the tests in the js/tests/redux folder.
* You can check in the console of the `edit post` page, by typing `wp.data.select( "yoast-seo/editor" )`.
* wp.data.select( "yoast-seo/editor" ).getTitleFallback("yoast-seo/editor") should get your snippet title.

## UI changes

## Documentation

## Quality assurance


Fixes #14384
